pkg/operator/podsecurityreadinesscontroller/podsecurityreadinesscontroller.go

```diff
@@ -0,0 +1,147 @@
+package podsecurityreadinesscontroller
+
+import (
+	"context"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	psapi "k8s.io/pod-security-admission/api"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	checkInterval = 10 * time.Minute // Adjust the interval as needed.
```

Comment:
in case of 10,000 namespaces I'd say ~10,000 API requests every 10 minutes is still quite a lot